lib/web/apiserver.go

```diff
@@ -308,6 +308,8 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 					if err == nil {
 						session.Session = base64.StdEncoding.EncodeToString(out)
 					}
+				} else {
+					log.WithError(err).Debugf("Could not authenticate.")
```

Comment:
Don't we stipulate that error messages should be in lower case?

Reply:
this is not an error message though, it's a debug log entry that is a proper english sentence. Error is a WithError fileld